dm/dm/proto/dmworker.proto

```diff
@@ -101,6 +101,11 @@ message CheckStatus {
 // DumpStatus represents status for dump unit
 // add fields later
 message DumpStatus {
+    int64 totalTables = 1;
```

Comment:
thanks for you work, could please also add this dump satus to OpenAPI ?

now we only have LoaderStatus and SycnerStatus in OpenAPI, you can do this work by following  these steps

1. add a new schema called`DumpStatus` in openapi yaml file [here](https://github.com/pingcap/ticdc/blob/3a2d2c5a5e9eec0eed946c5dfbdba3ce7210e526/dm/openapi/spec/dm.yaml#L1033)
2. use `make dm_generate_openapi` to generate stuct you added in yaml
3. add dump status in `openapi.go` like [loader/syncer](https://github.com/pingcap/ticdc/blob/3a2d2c5a5e9eec0eed946c5dfbdba3ce7210e526/dm/dm/master/openapi.go#L589) status 
4. add UT/IT for the new feature

btw you can open an issue to track this job and do this in next pr after this pr merged

Reply:
thanks for your suggestions, I am confident that I can finish this task